independent-projects/tools/codestarts/src/test/java/io/quarkus/devtools/codestarts/CodestartsTest.java

```diff
@@ -111,11 +113,13 @@ void checkSpecificProject() throws IOException {
 
         final Path targetDirectory = projectPath.resolve("specific-project");
         Codestarts.generateProject(codestartProject, targetDirectory);
-
+        System.out.println(targetDirectory.toAbsolutePath().toString());
```

Comment:
Is this a debugging leftover?

Reply:
Yes, good catch, since in tests I will remove it in the next PR :)